imagebuildah/stage_executor.go

```diff
@@ -316,8 +316,12 @@ func (s *StageExecutor) digestSpecifiedContent(node *parser.Node, argValues []st
 		} else {
 			// Source is not a URL, so it's a location relative to
 			// the all-content-comes-from-below-this-directory
-			// directory.
+			// directory.  Also raise an error if the src escapes
+			// the context directory.
 			contextSrc, err := securejoin.SecureJoin(contextDir, src)
+			if strings.HasPrefix(src, "../") && err == nil {
```

Comment:
nit:  Should reverse order and check if err == nil first, for speed, although maybe compiler is smart enough.

Reply:
touched up here and line 445

Reply:
just reversed here and at line 445